app/src/main/java/com/taskruler/MainViewModel.kt

```diff
@@ -92,7 +92,7 @@ class MainViewModel(var activityService : IActivityService = ActivityService())
         user?.let {
             user ->
             val document =
-            if (selectedUserTask.userTaskId == null || selectedUserTask.userTaskId.isEmpty()) {
+            if (selectedUserTask.userTaskId.isEmpty()) {
```

Comment:
Simplified is Boolean statement since you're comparing the exact same conditions together.